03.bowling/bowling.rb

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+score = ARGV[0]
+scores = score.split(',')
+
+shots = []
+scores.each do |s|
+  if s == 'X'
+    shots << 10
+    shots << 0
+  else
+    shots << s.to_i
+  end
+end
+
+frames = shots.each_slice(2).to_a
+
+point = 0
+frames.each_with_index do |frame, idx|
```

Comment:
```suggestion
point = frames.each_with_index.sum do |frame, idx|
```

のようにsumメソッドを使うとブロックの内部がよりシンプルになります

Reply:
修正しました。